test/integration/guest_user_test.rb

```diff
@@ -20,9 +21,9 @@ def setup
                             email: 'jwan622@example.com',
                             role: 0)
 
-    @item1 = Item.create(title: 'espresso', price: 9000)
+    @item1 = Item.create(title: 'espresso', description: "this is black gold", price: 30000)
     @category1 = item1.categories.create(name: 'Hot Beverages')
-    @item2 = Item.create(title: 'cold pressed coffee', price: 8000)
+    @item2 = Item.create(title: 'cold pressed coffee', price: 8000, description: "hipster nonsense")
```

Comment:
Line is too long. [100/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.